index.js

```diff
@@ -1,4 +1,4 @@
-const { compress: _compress, decompress: _decompress } = require('./bindings');
+const { compress: _compress, decompress: _decompress } = require('bindings')('zstd');
```

Comment:
Are we going to add typescript? 

Can we move this to a lib folder if not, or to src if so?

Reply:
For a two-function library probably not worth it :) and we can always add it later.